common/src/test/java/org/apache/sedona/common/S2Geography/WKTReadWriterTest.java

```diff
@@ -117,4 +113,18 @@ public void collection_roundTrip() throws ParseException {
     assertEquals(
         wkt, writeWithPrecision(reader.read(wkt), new PrecisionModel(PrecisionModel.FIXED)));
   }
+
+  @Test
+  public void empty_roundTrip() throws ParseException {
+    String point = "POINT EMPTY";
+    assertEquals(
+        point, writeWithPrecision(reader.read(point), new PrecisionModel(PrecisionModel.FIXED)));
+    String line = "LINESTRING EMPTY";
+    assertEquals(
+        line, writeWithPrecision(reader.read(line), new PrecisionModel(PrecisionModel.FIXED)));
+    String polygon = "POLYGON EMPTY";
+    assertEquals(
+        polygon,
+        writeWithPrecision(reader.read(polygon), new PrecisionModel(PrecisionModel.FIXED)));
+  }
```

Comment:
Can we add tests for `MULTIPOINT EMPTY`, `MULTILINESTRING EMPTY`, `MULTIPOLYGON EMPTY` and `GEOMETRYCOLLECTION EMPTY` for completeness?